src/Executor/ExecutionResult.php

```diff
@@ -153,7 +154,7 @@ public function toArray($debug = false)
             $result['data'] = $this->data;
         }
 
-        if (! empty($this->extensions)) {
+        if (count($this->extensions ?? []) > 0) {
```

Comment:
Sometimes `!== 0` is used and sometimes `> 0`. Is there a reason?

Reply:
No, just changed my mind midway through. Fixed in 44d6d20